### DISCUSSION TITLE
تطوير

### DISCUSSION THREAD

**Discussion by ibbrahiim** [community/community#83821] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

![Screenshot_2023-12-30-06-38-07-73](https://github.com/community/community/assets/153756238/e92c5b27-9aba-49ac-94a7-58fb5e8dda3a)

**Comment by Akash1134** [comment, marked as answer]
Hi @ibbrahiim , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
